src/components/color-system/examples/color-palette-extended.css

```diff
@@ -0,0 +1,308 @@
+/*
+ * To be able to use colors with alpha, they are written in RGB.
+ * Comments with HEX codes in front of the values are used as reference only.
+ * Example of use:
+ * solid: `color: rgb(var(--contrast-100))`
+ * transparent: `color: rgba(var(--contrast-100), 0.5)`
+ */
+
+:host {
```

Comment:
This file is just a copy of `lime-elements/src/style/color-palette-extended.css`, but with `:host` substituted for `:root` here and on line 169.

This fix is only needed until we enable dark-mode in the docs again, and the fix has been confined to a single commit, which can be reverted at that time.